Tentacle.xcodeproj/project.pbxproj

```diff
@@ -359,6 +504,84 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		61377C711C8A2B2C0081FF24 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				DEFINES_MODULE = YES;
+				DYLIB_COMPATIBILITY_VERSION = 1;
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Checkouts/Argo/build/Debug",
+					"$(PROJECT_DIR)/Carthage/Checkouts/Curry/build/Debug",
```

Comment:
Can you remove these from `Framework Search Paths` from `Build Settings` in `TentacleiOS`? Xcode adds them automatically, but they're unnecessary noise. :confused: